bytecomp/bytegen.ml

```diff
@@ -576,7 +576,7 @@ let rec comp_expr env exp sz cont =
         in
         comp_init env sz decl_size
       end
-  | Lprim((Pidentity | Popaque), [arg]) ->
+  | Lprim((Pidentity | Popaque | Pbytes_to_string | Pbytes_of_string), [arg]) ->
```

Comment:
this revision does do any change to the code gen, for both native and bytecode gen, but it is a  step to propagate it into bytecode or native backend, you can send another PR after this is merged